packages/styling/src/styles/theme.ts

```diff
@@ -125,9 +130,20 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean): ISema
     menuHeader: p.themePrimary,
 
     listBackground: p.white,
-    listTextColor: p.neutralPrimary,
+    listText: p.neutralPrimary,
     listItemBackgroundHovered: p.neutralLighter,
     listItemBackgroundChecked: p.neutralLight,
-    listItemBackgroundCheckedHovered: p.neutralQuaternaryAlt
+    listItemBackgroundCheckedHovered: p.neutralQuaternaryAlt,
+
+    // Deprecated slots, fixed by _fixDeprecatedSlots()
+    listTextColor: ''
   };
+
+  return toReturn;
 }
+
+function _fixDeprecatedSlots(s: ISemanticColors): ISemanticColors {
+  s.listTextColor = s.listText;
```

Comment:
What if the user passed in their own listTextColor but didn't pass in listText

Reply:
that's fine, they'd get a customized listTextColor but not a customized listText, as intended. hmm, i guess i could do it the other way and say listText: p.listTextColor || p.neutralPrimary........